client/components/themes-list/index.jsx

```diff
@@ -130,70 +139,148 @@ function ThemeBlock( props ) {
+		siteHasFeature( state, selectedSite?.ID, FEATURE_INSTALL_THEMES )
 	);
+	const canGoToPatternAssembler = useViewportMatch( 'large' );
 
+	// Prevents the Upwork upsell from being visible, so it doesn't conflict with the actions below.
 	useEffect( () => {
-		if ( shouldUpgradeToInstallThemes && ! emptyContent ) {
+		if ( selectedSite ) {
 			upsellCardDisplayed( true );
 		} else {
 			upsellCardDisplayed( false );
 		}
 		return () => {
 			upsellCardDisplayed( false );
 		};
-	}, [ emptyContent, shouldUpgradeToInstallThemes, upsellCardDisplayed ] );
+	}, [ selectedSite, upsellCardDisplayed ] );
 
-	if ( emptyContent ) {
-		return emptyContent;
+	if ( ! selectedSite ) {
+		return null;
 	}
 
-	return shouldUpgradeToInstallThemes ? (
-		<div className="themes-list__empty-container">
-			{ matchingThemes.length ? (
-				<WPOrgMatchingThemes
-					matchingThemes={ matchingThemes }
-					selectedSite={ selectedSite }
-					{ ...props }
-				/>
-			) : (
-				<>
-					<div className="themes-list__not-found-text">
-						{ translate( 'No themes match your search' ) }
+	return (
+		<div className="themes-list__footer">
+			<div className="themes-list__footer-heading">
+				{ translate( "Can't find what you're looking for?" ) }
+			</div>
+			<div className="themes-list__footer-subheading">
+				{ translate( 'Here are a few more options' ) }
+			</div>
+			<div className="themes-list__footer-action">
+				<Icon className="themes-list__footer-action-icon" icon={ addTemplate } size={ 28 } />
+				<div className="themes-list__footer-action-content">
+					<div className="themes-list__footer-action-text">
+						<div className="themes-list__footer-action-title">
+							{ translate( 'Create your own theme from scratch' ) }
+						</div>
+						<div className="themes-list__footer-action-description">
+							{ canGoToPatternAssembler
+								? translate(
+										'Start with a blank canvas and design your own homepage using our library of patterns.'
+								  )
+								: translate( 'Jump right into the editor to design your homepage from scratch.' ) }
+						</div>
+					</div>
+					<Button
+						primary
+						className="themes-list__footer-action-button"
+						href={
+							canGoToPatternAssembler
+								? `/setup/site-setup/patternAssembler?siteSlug=${ selectedSite.slug }&backTo=/themes/${ selectedSite.slug }`
+								: `/site-editor/${ selectedSite.slug }`
+						}
+					>
+						{ canGoToPatternAssembler
+							? translate( 'Start designing' )
+							: translate( 'Open the editor' ) }
+					</Button>
+				</div>
+			</div>
+			<div className="themes-list__footer-action">
+				<Icon className="themes-list__footer-action-icon" icon={ brush } size={ 28 } />
+				<div className="themes-list__footer-action-content">
+					<div className="themes-list__footer-action-text">
+						<div className="themes-list__footer-action-title">
+							{ translate( 'Hire our team of experts to design one for you' ) }
```

Comment:
All of the other titles explicitly say 'theme': "Create your own theme from scratch", "Upload your theme". Should this say "Hire our team of experts to design a theme for you"?

Reply:
Actually maybe this is intentionally vague, lets leave it.

DIFM doesn't design a theme for you. You have to choose a theme from the even more limited list offered, or let us choose one of them for you. DIFM will then make custom page layouts for you.

Reply:
> Actually maybe this is intentionally vague, lets leave it.

You made a good point about being vague though, since translators for example might have trouble understanding what "one" means in this context, so we can probably add a translator comment to clarify.

Reply:
+1 for being vague and +1 for adding a translator comment! 👍